auth/cmd/cmd.go

```diff
@@ -60,6 +62,7 @@ func FlagSet() *pflag.FlagSet {
 	flags.Int(ConfAccessTokenLifeSpan, defs.AccessTokenLifeSpan, "defines how long (in seconds) an access token is valid. Uses default in strict mode.")
 	flags.StringSlice(ConfContractValidators, defs.ContractValidators, "sets the different contract validators to use")
 	flags.Bool(ConfV2APIEnabled, defs.V2APIEnabled, "enables experimental v2 API endpoints")
+	flags.String(ConfPresentationExchangeMappingFile, defs.PresentationExchangeMappingFile, "sets the path to the presentation exchange mapping file")
```

Comment:
Do we want to document it (the feature, file format, etc) at this point, for ourselves and others, or is it probable that it'll change in (near) future?

Reply:
marked is as hidden for now. We'll demo first and then decide on the next step